script.js

```diff
@@ -122,7 +122,7 @@ let deletePost = async function (domNode) {
 
     if (debug) console.log("Deleted successfully");
 
-    await delay(200);
+    await delay(3000);
```

Comment:
Basically all timestamps should be moved to constants and referenced from there instead of having them sprinkled I suppose. I will make that change in a future version. This script was a quick hack in the middle of the night to clear out a group.